src/Core/Commander/Providers/TiledImageTools.js

```diff
@@ -0,0 +1,126 @@
+import * as THREE from 'three';
+import Fetcher from './Fetcher';
+import CacheRessource from './CacheRessource';
+import IoDriver_XBIL from './IoDriver_XBIL';
+import Projection from '../../Geographic/Projection';
+import CoordWMTS from '../../Geographic/CoordWMTS';
+import { UNIT } from '../../Geographic/Coordinates';
+
+const SIZE_TEXTURE_TILE = 256;
+
+// CacheRessource is necessary for neighboring PM textures
+// Info : THREE.js have cache image https://github.com/mrdoob/three.js/blob/master/src/loaders/ImageLoader.js#L25
+const cache = CacheRessource();
+const ioDXBIL = new IoDriver_XBIL();
+const projection = new Projection();
+
+const _cropXbilTexture = function _cropXbilTexture(texture, pitch) {
```

Comment:
Why? It makes no sense to do that. `TiledImageTools` is not a promise factory. Factorization is only good when it's useful. And it's not useful to avoid a `Promise.resolve`. Adding a method just for that contribute to make the code harder to read and to maintain.